### PR TITLE
add script for aws-opsworks-smooth-deploy

### PR DESCRIPTION
**Pull request by rifki192**

### CODE REVIEW

**Comment by rifki192** [comment]
@petrabarus please review

File: opsworkssmoothdeploy/deploy.py [hunk 1]
```diff
@@ -14,61 +14,206 @@
...
+            },
+        ]
+    )
     pass
 
-def detach(opsworks_stack_id):
+def detach(stack_id, minimum_instance = __MINIMUM_ATTACHED_INSTANCES__):
     """Detach current instance from the load balancer if the active instance
     is still in threshold.
     :param opsworks_stack_id The ID of the opsworks stack.
```
Comment (petrabarus) [inline opsworkssmoothdeploy/deploy.py:179]: forgot the doc for minimum instance

**Comment by petrabarus** [comment]
squash the commit

**Comment by petrabarus** [comment]
lgtm. let me know when the commits squashed.

**Pull request merged**